src/components/interactive-builder/add-question-modal.component.tsx

```diff
@@ -580,6 +604,76 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
                   />
                 </RadioButtonGroup>
               ) : null}
+
+              {questionType === 'programState' && (
+                <>
+                  <div>
+                    <TextInput
+                      id="program"
+                      labelText="Add program uuid"
```

Comment:
Can we fetch these programs from the backend and render them in a ComboBox or similar? I wouldn't expect the user to know which UUIDs exist ahead of time.